hdlparse/vhdl_parser.py

```diff
@@ -6,748 +6,829 @@
 import os
 import re
 from pprint import pprint
-
 from hdlparse.minilexer import MiniLexer
 
-"""VHDL documentation parser"""
+'''VHDL documentation parser'''
```

Comment:
Why are double quotes replaced by single quotes?

This will give linting warnings.

Reply:
@vvvverre used PyCharm to change this and a bunch of indenting, but nothing functional. I simply switched the file back to Andres's branch due to the functionality enhancement, but left the imports python3. Maybe I can try reproducing @vvvverre's work with PyCharm. If you have any linters you recommend, I can look into fixing that before merge.

Reply:
@umarcor and me have developed over time a quite enhanced pipeline with a lot of checks for Python code.

![image](https://user-images.githubusercontent.com/956109/143775063-4a5ad5c1-baf1-4a7a-9bfa-87852e515473.png)  
Source: https://github.com/pyTooling/pyTooling/actions/runs/1512395427

It uses:
* unit tests with pytest
* branch coverage with coverage.py
* static type checking with mypy
* further checks (icl. linting) via Codacy
  * Bandit
  * Propspector
  * Pylint
  * remark-lint
  * (other can be activated)
  
@umarcor currently generalizes these jobs into reusable workflows, so it becomes easier to reuse such a complex workflow.

Reply:
Awesome, I will look into it.

Reply:
@michael-etzkorn see pyTooling/Actions#1. Not ready to use yet, but we hope to merge in the following days.
I will then update #2 to use the reusable workflows, and might update this as well.

Reply:
Will be handled via #2.